app/src/main/java/ch/epfl/favo/common/DocumentCallback.java

```diff
@@ -0,0 +1,7 @@
+package ch.epfl.favo.common;
+
+import java.util.Map;
+
+public interface DocumentCallback {
```

Comment:
I realize having `DocumentCallback` and `MultipleDocumentsCallback` is not exactly the best way to do this. Need some advice here.

Reply:
Something else I had in mind was just keeping `MultipleDocumentsCallback` and removing `DocumentCallback`. This would mean that the callback for `getDocument` would be an `ArrayList` of just one element, but this seems even worse. Let me know if you guys have any other ideas!